expr/expr_test.go

```diff
@@ -727,11 +727,10 @@ func a(t *testing.T) {
 	// Test casts to time
 	ts := zng.Value{zng.TypeTime, zng.EncodeTime(nano.Ts(1589126400_000_000_000))}
 	testSuccessful(t, "1589126400.0 :time", nil, ts)
+	ts = zng.Value{zng.TypeTime, zng.EncodeTime(nano.Ts(1589126400))}
 	testSuccessful(t, "1589126400 :time", nil, ts)
-	testError(t, `"1234" :time`, nil, expr.ErrBadCast, "cannot cast string to time")
-}
+	testSuccessful(t, `"1589126400" :time`, nil, ts)
```

Comment:
@mccanne: Does this part look OK to you?

Reply:
I think so.